<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BanqueWidget</class>
 <widget class="QWidget" name="BanqueWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BanqueWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #101010</string>
  </property>
  <widget class="QPushButton" name="pushButton_Connexion">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #151515; font-size: 24px; color: #ffffff; font-weight: bold;</string>
   </property>
   <property name="text">
    <string>Connexion</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_Informations">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>611</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #151515; font-size: 12px; color: #808080; font-weight: bold;</string>
   </property>
   <property name="title">
    <string>Informations de la banque</string>
   </property>
   <widget class="QTextEdit" name="textEdit_Informations">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>591</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #151515; font-size: 12px; color: #808080; font-weight: bold;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Informations">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>281</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #151515; font-size: 12px; color: #808080; font-weight: bold;</string>
    </property>
    <property name="text">
     <string>Afficher les informations de tous les comptes</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_Informations">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>591</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #151515; font-size: 12px; color: #808080; font-weight: bold;</string>
    </property>
    <column>
     <property name="text">
      <string>Numéro</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nom</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Prénom</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Solde (€)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Agence</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ville</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_Quitter">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #151515; font-size: 24px; color: #ffffff; font-weight: bold;</string>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Ajouter">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #151515; font-size: 24px; color: #808080; font-weight: bold;</string>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Quitter</sender>
   <signal>clicked()</signal>
   <receiver>BanqueWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>526</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
